draft-ietf-scitt-architecture.md

```diff
@@ -568,14 +568,10 @@ Multi-tenant support can be enabled through the use of identifiers in the `iss`
 
 ### Registration Policies
 
-Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is accepted to be registered to the Verifiable Data Structure.
+Registration Policies refer to additional checks over and above the Mandatory Registration Checks that are performed before a Signed Statement is registered to the Verifiable Data Structure.
+For consistency across implementations, Transparency Services MUST maintain Registration Policies.
```

Comment:
Why? The policies themselves are not defined or interoperable. Do we have to define an 'empty' policy as the lowest common denominator?